pkg/services/searchV2/service.go

```diff
@@ -0,0 +1,106 @@
+package searchV2
+
+import (
+	"bytes"
+	"context"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+	"github.com/grafana/grafana/pkg/services/searchV2/extract"
+	"github.com/grafana/grafana/pkg/services/sqlstore"
+)
+
+type StandardSearchService struct {
+	sql *sqlstore.SQLStore
+}
+
+func ProvideService(sql *sqlstore.SQLStore) SearchService {
```

Comment:
I tried sticking this in the existing search service.... but circular dependencies on `sqlstore` made that very difficult :(